compiler/src/diagnostics/graindoc_parser.mly

```diff
@@ -0,0 +1,48 @@
+%token PARAM SECTION SINCE HISTORY THROWS RETURNS EXAMPLE DEPRECATED COLON EOL EOF
+%token <string> TEXT IDENT SEMVER CONSTRUCTOR
+
+%right EOL
+
+%start <(Comment_attributes.parsed_graindoc)> graindoc
```

Comment:
Can this just be `(option(string), list(t))`
```suggestion
%start <((option(string), Comments.Attribute.attributes))> graindoc
```

Reply:
I tried that originally, but it's a syntax error. Seems like you have to provide a named type here.